bedrock/firefox/templates/firefox/hello/start-38.1.html

```diff
@@ -0,0 +1,20 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "/firefox/hello/start.html" %}
```

Comment:
NIT: Technically it works, but paths in the `extends` tag shouldn't start with a `/`.